python/adapters/s3_storage_adapter.py

```diff
@@ -177,11 +143,11 @@ def __test_connection(self):
             self.get_last_file_sequence()
             # Trying to write 'test' ledger, then delete it. Not using the 'save' method, as it will rewrite last_file
             self.file_name = 'test'
-            self._save_creations([{'source': 'GCQTAWULBNFLBAEQLEN6FDGGCPYTVZ3Y55AB4F7HSTMQKNX3HZINMQJM',
-                                   'destination': 'GDDFYG3OSTSHADS7SP6TZ4XM62EQ522CI7UYJSNAETGJJCGOX66TP5Q5',
+            self._save_operations([{'source': 'JUST_A_TEST',
```

Comment:
doesn't really matter, but I would use a real address instead of some string, cant know what absurd bug can show up